_posts/patterns/2016-01-28-alerts.md

```diff
@@ -0,0 +1,275 @@
+---
+layout:         pattern
+title:          Alerts
+date:           2015-01-28 00:00:00
+
+categories:     patterns
+tags:
+- atomic
+- pattern
+- alerts
+
+slug:           alerts
+
+url_styles:     "patterns/_alerts.scss"
+
+description:    Examples of alerts in a semi-real world context.
+
+info:           Alerts provide information about...TBD
+---
+
+<h3 class="hd-6 example-set-hd">Error w/ Title</h3>
+<div class="example-set">
+    <div class="alert alert-error">
+        <div class="icon-display">
```

Comment:
Hmm, looks like you're following/copying from our example icon markup here with the ``<div class="icon-display">``. This element isn't needed to display icons and can be removed. I'd rely on just styling the ``.alert-icon`` element you have nested in here.